DEMS-Common/test/UDP/SocketTest.java

```diff
@@ -38,8 +38,12 @@
     private Thread m_ListenerThread;
     Message msg;
 
+    public static int TEST_PORT = 35646;
+    public static String TEST_ADDR = "239.1.1.1";
```

Comment:
Should it be localhost? Here 127.0.0.1

Reply:
Nop all the communication will take place over UDP as a simplification. This way there's no hassle over the IP address and the entries in the AddressBook